src/main/java/com/owncloud/android/lib/resources/files/ChunkedFileUploadRemoteOperation.java

```diff
@@ -181,7 +182,7 @@ protected RemoteOperationResult run(OwnCloudClient client) {
             if (token != null) {
                 moveMethod.addRequestHeader(E2E_TOKEN, token);
             }
-            int moveResult = client.executeMethod(moveMethod);
+            int moveResult = client.executeMethod(moveMethod, CalcAssembleTimeout(file), -1);
```

Comment:
For -1 can you introduce a const, e.g. DO_NOT_CHANGE_DEFAULT, so that when reading the code one directly knows that this value does not change connection timeout.